README.md

```diff
@@ -1,18 +1,22 @@
-# XMTP Example
+# XMTP React Native quickstart example app
 
-This example shows you how to create a basic xmpt react native app that calls through to `xmtp-js` via a webview. We are working towards making this more full featured and you can follow the progress here: https://github.com/xmtp/xmtp-quickstart-react-native/issues/2
+Use this quickstart example app to learn how to create a basic XMTP React Native app that calls through to [`@xmtp/react-native-sdk`](https://github.com/xmtp/xmtp-react-native).
 
-***Note: This is necessary because the JS engine React Native uses (Hermes) does not work with our crypto and polyfills (you can see an example using that here: https://github.com/xmtp/example-chat-react-native)***
+## Run the example app 🚀
 
-Long term we plan to have a React Native SDK that will be easy to integrate with and also performant. You can follow the progress of that here: https://github.com/xmtp/xmtp-react-native/issues/11
+1. Run `npm install`.
+2. Run `npm run start`.
+3. Wait until the app is built and downloaded. Press "r" to reload the app and splash screen.
```

Comment:
@nplasterer thank you so much for this PR! I've made a few updates - tech writer stuff! =)

May I ask - when I ran through these steps - I wasn't able to get the apps running. I'm pretty sure it is user error! 

For example, when I see the Metro screen, I press "i". My ios simulator opens and then I get a BUILD FAILED. I tried pressing "a" for android and no simulator opens and I get a BUILD FAILED. I wonder if I'm missing a prereq? =)

Reply:
I think this should be `npm run android` or `npm run ios` to make sure it boots